en_ewt-ud-dev.conllu

```diff
@@ -14210,7 +14210,7 @@
 15	in	in	ADP	IN	_	16	case	16:case	_
 16	protest	protest	NOUN	NN	Number=Sing	8	obl	8:obl:in	_
 17	at	at	SCONJ	IN	_	23	mark	23:mark	_
-18	how	how	ADV	WRB	PronType=Int	23	advmod	23:advmod	_
+18	how	how	SCONJ	WRB	PronType=Int	23	mark	23:mark	_
```

Comment:
P + SCONJ! I guess this is the correct analysis

Reply:
yes, mark + SCONJ IMO